.github/workflows/update-translation.yml

```diff
@@ -0,0 +1,57 @@
+name: Update Translation Catalog
+on: 
+    push:
+      branches: 
+       - main
+      paths:
+        - src/**
```

Comment:
Maybe specify libslic3r and slic3r with files that end in .cpp/.hpp? Could save a few unnecessary runs.